altair/utils/core.py

```diff
@@ -14,7 +14,7 @@
 import pandas as pd
 import numpy as np
 
-from altair.utils.schemapi import SchemaBase, Undefined
+from altair.utils.schemapi import SchemaBase, Undefined  # noqa: F401
```

Comment:
Why do we need this? If this PR removed the only place where `Undefined` was used, shouldn't we remove the import instead?

Reply:
Thanks for checking on this @joelostblom.  I left the `Undefined` import here because of this line: https://github.com/altair-viz/altair/blob/7bc754e8dd6748a8b4633b6f3cd2345ab7078fb2/altair/utils/__init__.py#L10

Do you think it's worth importing `Undefined` directly in that `__init__` file instead?  I haven't experimented at all, so it's possible other errors will show up if I move it.

Reply:
Yes that would make more sense to me. Could you try that? If it doesn't work and seems laborious to resolve we can just keep it here, but let's give it a quick try first.

Reply:
That seemed to work @joelostblom in https://github.com/altair-viz/altair/pull/2841/commits/c91468f53d9a2d08dfa6a84790d860a1d970ad04
My quick tests did not catch any unwanted side effects.